### PR TITLE
Enh collection zxtm pool

### PR DESCRIPTION
**Pull request by DalfoCentreon**
## Description

New snmp collection files to monitor pools a virtual servers of ZXTM devices.
https://www.circitor.fr/Mibs/Html/Z/ZXTM-MIB-SMIv2.php

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [x] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [x] 21.10.x
- [x] 22.04.x
- [x] 22.10.x
- [x] 23.04.x
- [x] 23.10.x (master)

#### Community contributors & Centreon team

- [ ] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: src/contrib/collection/snmp/zxtm-virtualserver.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                    "nlabel": "conn",
+                    "instances": ["%(virtualserverEntry.virtualserverName)"],
+                    "value": "%(virtualserverEntry.virtualserverCurrentConn)",
+                    "critical": "%(constants.criticalConn)",
+                    "unit": "conn",
+                    "min": 0
+                }
+            ],
+            "formatting": {
+                "printf_msg": "Device '%s' current connetions are '%d', max connections are '%d', total HTTP requests are '%d'",
```
Comment (garnier-quentin) [inline src/contrib/collection/snmp/zxtm-virtualserver.json:53]: should be ```connections``` ?

File: src/contrib/collection/snmp/zxtm-virtualserver.json [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                        "name": "virtualserverTotalHTTPRequests",
+                        "oid": ".1.3.6.1.4.1.7146.1.2.2.2.1.43"
+                    }
+                ]
+            }
+        ]
+    },
+    "selection_loop": [
+        {
+            "name": "Virtual SErver ZXTM",
```
Comment (garnier-quentin) [inline src/contrib/collection/snmp/zxtm-virtualserver.json:36]: Should be ```Server``` ?

File: src/contrib/collection/snmp/zxtm-virtualserver.json (hunk 1 above)
Comment (DalfoCentreon) [reply src/contrib/collection/snmp/zxtm-virtualserver.json:53]: Yes, it is a typo.

File: src/contrib/collection/snmp/zxtm-virtualserver.json (hunk 2 above)
Comment (DalfoCentreon) [reply src/contrib/collection/snmp/zxtm-virtualserver.json:36]: Yes, it is a typo.

**Comment by garnier-quentin** [comment]
@DalfoCentreon it's not corrected in the PR

**Review by bouda1 — APPROVED** [review]

**Pull request merged**